modules/passkey/contracts/SafeWebAuthnSignerProxyHarness.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
move this file to certora folder under helpers

Reply:
moved under harnesses